src/setup-dotnet.ts

```diff
@@ -8,7 +9,20 @@ async function run() {
     // Version is optional.  If supplied, install / use from the tool cache
     // If not supplied then task is still used to setup proxy, auth, etc...
     //
-    const version = core.getInput('version');
+    let version = core.getInput('version');
```

Comment:
Const, add type

Reply:
Not a const since we change the value below, added type